setup.py

```diff
@@ -212,22 +213,21 @@ def get_llvm_package_info():
         elif arch == 'x64':
             vglibc = tuple(map(int, platform.libc_ver()[1].split('.')))
             vglibc = vglibc[0] * 100 + vglibc[1]
-            if vglibc > 228:
-                # Ubuntu 24 LTS (v2.39)
-                # Ubuntu 22 LTS (v2.35)
-                # Ubuntu 20 LTS (v2.31)
+            # NOTE: need to update these values when we
+            # have new precompiled llvm builds
+            if vglibc > 233:
+                # llvm-f6ded0be-ubuntu-x64 requires at least glibc 2.34
```

Comment:
Isn't this llvm hash frequently changed? Might be better to just to have a more configurable way to update the version numbers (i.e., 233/226/215). For example, document the numbers somewhere in a file

Reply:
I've now updated the code to use `*.txt` files under `cmake/llvm-vglibc` to determine the glibc version numbers, and we can use `update.sh` scripts to update them.